resources/core/charts/k8s-dashboard-proxy/templates/deployment.yaml

```diff
@@ -0,0 +1,105 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: {{ template "name" . }}
+  namespace: {{ .Release.Namespace }}
+spec:
+  {{ if not .Values.global.isLocalEnv }}
+  replicas: 2
+  strategy:
```

Comment:
Looks like a copy paste from `apiserver-proxy`. Do you need 2 replicas for this deployment?

Reply:
Yeah, I started from the `apiserver-proxy` to create mine. Not sure about the 2 replicas, my idea was the HA in production mode, what do you suggest?

Reply:
I'd prefer going for the HA in production and a single instance in the local installation.

Reply:
We don't need to have 2 replicas here. This is only used by an "admin" and he can wait till K8S is restarting the pod, if something is happen.

Reply:
OK fine